### PR TITLE
Add uwsgi service file 

### PR DESCRIPTION
**Pull request by aazam-gh**
A uwsgi.service file added under /lib/systemd/system for starting feilong in uwsgi

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -69,5 +69,6 @@
                 ('/var/lib/zvmsdk', ['data/setupDisk']),
                 ('/etc/sudoers.d', ['data/sudoers-zvmsdk']),
                 ('/etc/zvmsdk', ['data/uwsgi-zvmsdk.conf']),
-                ('/etc/zvmsdk', ['doc/source/zvmsdk.conf.sample'])],
+                ('/etc/zvmsdk', ['doc/source/zvmsdk.conf.sample']),
+                (['data/uwsgi.service'])],
```
Comment (Bischoff) [inline setup.py:73]: No, it should be similar to line 68, as it goes to `/lib/systemd/system`.

Move it up there, and add the destination path.

File: data/uwsgi.service [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[Unit]
```
Comment (Bischoff) [inline data/uwsgi.service:1]: This file will conflict with the service file for `uwsgi` when you install it.

I was thinking at a file named `zvmsdk-wsgi.service`, like the one in documentation section 4.2.2:

https://cloudlib4zvm.readthedocs.io/en/latest/setuphttpd.html#start-feilong-in-uwsgi

File: data/uwsgi.service [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Description=Feilong uwsgi
+After=syslog.target network.target httpd.service
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/uwsgi --ini /data/uwsgi-zvm.conf
```
Comment (Bischoff) [inline data/uwsgi.service:7]: The configuration file is in `/etc/zvmsdk/uwsgi-zvmsdk.conf`.

**Review by Bischoff — CHANGES_REQUESTED** [review]
See comments

**Comment by aazam-gh** [comment]
Addressed the changes

File: data/zvmsdk-wsgi.service [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+After=syslog.target network.target httpd.service
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/uwsgi --ini /etc/zvmsdk/uwsgi-zvmsdk.conf
+ExecReload=/usr/sbin/uwsgi --reload /etc/zvmsdk/uwsgi-zvmsdk.conf
+ExecStop=/usr/sbin/uwsgi --stop /etc/zvmsdk/uwsgi-zvmsdk.conf
+
+[Install]
+WantedBy=multi-user.target
```
Comment (Bischoff) [inline data/zvmsdk-wsgi.service:12]: nitpick: add missing end of line

**Review by Bischoff — COMMENTED** [review]
thanks for the changes

added a last nitpick

will test it live now

**Review by Bischoff — APPROVED** [review]
tested to work fine

**Review by bjhuangr — APPROVED** [review]

**Comment by bjhuangr** [comment]
@aazam-gh please sign your PR, thanks

**Pull request merged**